common/djangoapps/student/helpers.py

```diff
@@ -40,6 +41,8 @@ def check_verify_status_by_course(user, course_enrollments):
         * VERIFY_STATUS_NEED_TO_VERIFY: The student has not yet submitted photos for verification.
         * VERIFY_STATUS_SUBMITTED: The student has submitted photos for verification,
           but has have not yet been approved.
+        * VERIFY_STATUS_RESUBMITTED: The student has re-submitted photos for re-verification,
```

Comment:
`The student has re-submitted photos for re-verification while they still have an active but expiring ID verification`